xml/ha_migration.xml

```diff
@@ -411,8 +449,6 @@
       <para>
        Test the upgrade procedure on a staging instance of your cluster
        setup first, before performing it in a production environment.
```

Comment:
Is the comma here correct? IHMO it is not needed, but I'm not sure.

Reply:
Let's leave this up to the proofreaders. :)